modules/services/sxhkd.nix

```
@@ -73,6 +73,8 @@ in
     ];
 
     xsession.initExtra = ''
+      [[ $(systemctl is-active --user sxhkd.scope) == active ]] \
+        && systemctl --user stop sxhkd.scope
```

```suggestion
      systemctl --user stop sxhkd.scope 2> /dev/null || true
```

This runs into a race condition where the scope is stopped between the two `systemctl` calls.

Don't check, handle failure.